nixos/release.nix

```diff
@@ -303,6 +303,7 @@ in rec {
   #tests.panamax = hydraJob (import tests/panamax.nix { system = "x86_64-linux"; });
   tests.peerflix = callTest tests/peerflix.nix {};
   tests.postgresql = callSubTests tests/postgresql.nix {};
+  tests.postgis = callTest tests/postgis.nix { system = "x86_64-linux"; };
```

Comment:
Do you have any particular reason to restrict to "x86_64" ? Why not letting it open:

```
tests.postgis = callTest tests/postgis.nix {};
```

Reply:
If the next push is ok (test, ok for you) i'll rebase